test/test_installation.py

```diff
@@ -97,6 +97,7 @@ def _create_match_list(self) -> List[str]:
             f"lib/cmake/umf/umf-targets-{self.build_type}.cmake",
             "lib/cmake/umf/umf-targets.cmake",
         ]
+        lib.append(f"lib/{lib_prefix}coarse.{lib_ext_static}")
```

Comment:
Could you please explain this line? Do we want to install `coarse` lib? if it is an internal lib should we really install it?

Reply:
We should not. Fixed.
Done.